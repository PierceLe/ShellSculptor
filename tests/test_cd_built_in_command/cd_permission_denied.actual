>> cd /root
cd: no such file or directory: /root
>> pwd
/Users/piercele/Desktop/year-2/sem1/info1112/ass1/ShellSculptor/tests

