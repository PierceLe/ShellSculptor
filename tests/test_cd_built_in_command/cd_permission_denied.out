>> cd /root
cd: permission denied: /root
>> pwd
/home/tests

